test/e2e/framework/volume/fixtures.go

```diff
@@ -87,6 +87,12 @@ const (
 	iSCSIIQNTemplate = "iqn.2003-01.io.k8s:e2e.%s"
 )
 
+// SizeRange encapsulates a range of sizes specified as minimum and maximum quantity strings
+type SizeRange struct {
+	Max string // Max quantity specified as a string including units. E.g "1Gi"
+	Min string // Min quantity specified as a string including units. E.g "3Gi"
```

Comment:
Super nit: Curious about the E.g of `Min` is bigger than one of `Max` ;-)

Reply:
Well, it may fail to get a suitable size, depend on the case. We don't validate it and assume that is valid. Should I considerate `Skipf` or `Failf`？

Reply:
Sorry for my unclear comment. 
I just wanted to say it is a little strange to put the `E.g` of Min which is bigger than one of Max.
How about
```
	Max string // Max quantity specified as a string including units. E.g "3Gi"
	Min string // Min quantity specified as a string including units. E.g "1Gi"
```
instead?

Reply:
@oomichi good catch. Yes, that makes sense.